cardano-cli/src/Cardano/CLI/EraBased/Options/Genesis.hs

```diff
@@ -436,3 +435,5 @@ pBulkPoolsPerFile =
     , Opt.help "Each bulk pool to contain this many pool credential sets [default is 0]."
     , Opt.value 0
     ]
+
```

Comment:
Random diff

Reply:
There was an unnecessary empty line on 412. Just deleted that empty line pulling everything bellow that one line up.